registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidator.java

```diff
@@ -113,6 +116,18 @@
 
 	@Value("${registration.processor.applicant.dob.format}")
 	private String dobFormat;
+	
+	@Value("${packetmanager.provider.osivalidator.parentOrGuardianBiometrics}")
+	private String parentOrGuardianBiometricsProvider;
+	
+	@Value("${packetmanager.provider.osivalidator.officerBiometricFileName}")
+	private String officerBiometricFileNameProvider;
+	
+	@Value("${packetmanager.provider.osivalidator.supervisorBiometricFileName}")
+	private String supervisorBiometricFileNameProvider;
+	
+	@Value("${registration.processor.osi.sub-processes}")
+	private String subProcesses;
```

Comment:
common property should be used. the subprocess is common for all stages. Each stage wont have its own property for subprocess. Talk to mamta and use common sub process